ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/TopicNode.java

```diff
@@ -63,9 +65,9 @@
   Topic build(final String defaultFormat) {
     final String formatToUse = format.replace("{FORMAT}", defaultFormat);
 
-    final SerdeSupplier<?> keySerdeSupplier = SerdeUtil.getSerdeSupplier(
-        Format.KAFKA,
-        this::logicalSchema
+    final SerdeSupplier<?> keySerdeSupplier = SerdeUtil.getKeySerdeSupplier(
+        KeyFormat.nonWindowed(FormatInfo.of(Format.DELIMITED)),
```

Comment:
why are we changing the key from KAFKA to DELIMITED? Also, looking at `getSerdeSupplier` this does the same thing as the previous code because we never pass in a windowed key here so why make this change at all?

Reply:
This is just a place holder. Indeed I changes it from DELIMITED to string serde supplier.
The correct serdes will be set when we analyze the statements.